docs/settings.rst

```diff
@@ -596,7 +596,11 @@ Enables EDNS subnet processing, for backends that support it.
 -  Boolean
```

Comment:
not a boolean :)

I'd say 'string', 'ne of: no, yes, shared'

Reply:
amended